WordPress/Classes/Networking/SharingServiceRemote.swift

```diff
@@ -0,0 +1,297 @@
+                    conn.externalDisplay = dict.stringForKey(ConnectionDictionaryKeys.externalDisplay) ?? conn.externalDisplay
+                    conn.externalID = dict.stringForKey(ConnectionDictionaryKeys.externalID) ?? conn.externalID
+                    conn.externalName = dict.stringForKey(ConnectionDictionaryKeys.externalName) ?? conn.externalName
+                    conn.externalProfilePicture = dict.stringForKey(ConnectionDictionaryKeys.externalProfilePicture) ?? conn.externalProfilePicture
+                    conn.keyringID = dict.numberForKey(ConnectionDictionaryKeys.ID) ?? conn.keyringID
+                    conn.label = dict.stringForKey(ConnectionDictionaryKeys.label) ?? conn.label
+                    conn.refreshURL = dict.stringForKey(ConnectionDictionaryKeys.refreshURL) ?? conn.refreshURL
+                    conn.status = dict.stringForKey(ConnectionDictionaryKeys.status) ?? conn.status
+                    conn.service = dict.stringForKey(ConnectionDictionaryKeys.service) ?? conn.service
+                    conn.type = dict.stringForKey(ConnectionDictionaryKeys.type) ?? conn.type
+                    conn.userID = dict.numberForKey(ConnectionDictionaryKeys.userID) ?? conn.userID
+
+                    return conn
+                }
+
+                onSuccess(keyringConnections)
+            },
+            failure: { (operation: AFHTTPRequestOperation!, error: NSError!) -> Void in
+                failure?(error)
+        })
+    }
+
+
+    /**
+     *  @brief      Fetches the current user's list of Publicize connections for the specified site's ID.
+     *
+     *  @param      siteID      The WordPress.com ID of the site.
+     *  @param      success     An optional success block accepting an array of `RemotePublicizeConnection` objects.
+     *  @param      failure     An optional failure block accepting an `NSError` argument.
+     */
+    public func getPublicizeConnections(siteID:NSNumber, success: ([RemotePublicizeConnection] -> Void)?, failure: (NSError! -> Void)?) {
+        let endpoint = "sites/\(siteID)/publicize-connections"
+        let path = self.pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+
+        api.GET(path,
+            parameters: nil,
+            success: {(operation:AFHTTPRequestOperation!, response:AnyObject!) -> Void in
+                guard let onSuccess = success else {
+                    return
+                }
+
+                let responseDict = response as! NSDictionary
+                let connections:Array = responseDict.arrayForKey(ConnectionDictionaryKeys.connections)
+                let publicizeConnections:[RemotePublicizeConnection] = connections.map { (let dict) -> RemotePublicizeConnection in
+                    let conn = self.remotePublicizeConnectionFromDictionary(dict as! NSDictionary)
+                    return conn
+                }
```

Comment:
(No need to change anything sir!). 

```
connections.map {
    return self.remotePublicizeConnectionFromDictionary($0 as! NSDictionary)
}
```

But maybe that's a little bit _less_ legible!